client-src/elements/chromedash-all-features-page.ts

```diff
@@ -15,37 +28,34 @@ export class ChromedashAllFeaturesPage extends LitElement {
     ];
   }
 
-  static get properties() {
-    return {
-      rawQuery: {type: Object},
-      title: {type: String},
-      showQuery: {type: Boolean},
-      query: {type: String},
-      columns: {type: String},
-      showEnterprise: {type: Boolean},
-      sortSpec: {type: String},
-      user: {type: Object},
-      start: {type: Number},
-      num: {type: Number},
-      starredFeatures: {type: Object},
-      selectedGateId: {type: Number},
-    };
-  }
+  @property({type: String})
+  title = 'Features';
+  @property({type: Boolean})
+  showQuery = true;
+  @property({type: Object})
+  user: User = null!;
+  @property({type: Number})
+  selectedGateId = 0;
 
-  constructor() {
-    super();
-    this.title = 'Features';
-    this.showQuery = true;
-    this.query = '';
-    this.columns = 'normal';
-    this.sortSpec = '';
-    this.showEnterprise = false;
-    this.user = {};
-    this.start = 0;
-    this.num = 100;
-    this.starredFeatures = new Set();
-    this.selectedGateId = 0;
-  }
+  @state()
+  rawQuery: RawQuery | undefined = {};
```

Comment:
This needs to be an `@property` since it's set from outside the element, at https://github.com/GoogleChrome/chromium-dashboard/blob/9bff5bc3b05d091b5f1e7270175091222ba72aa9/client-src/elements/chromedash-app.js#L343. Double-check the other fields too.